docs/devguide/migrate-dashboards.asciidoc

```diff
@@ -93,6 +93,6 @@ Using the yml file, you can export all the dashboards for a single module or for
 [source,shell]
 ----
 cd metricbeat/module/system
-go run ../../../dev-tools/cmd/dashboards/export_dashboards.go -yml module.yml
+go run ../../../dev-tools/cmd/dashboards/export_dashboards.go --yml module.yml
```

Comment:
Up in this file there is a hit for `-dashboard` that should be `--dashboard`.

Reply:
go catch.  Actually it should be `-yml` and `-dashboard` is correct.  the `export_dashboards.go` is a separate go program that uses `flags` from stdlib, so the one dash arg is correct.

Updated.